public/app/features/explore/LogsMetaRow.tsx

```diff
@@ -78,7 +78,7 @@ export const LogsMetaRow: React.FC<Props> = React.memo(
           >
             <Button variant="secondary" size="sm" onClick={onEscapeNewlines}>
               <span>{forceEscape ? 'Remove escaping' : 'Escape newlines'}&nbsp;</span>
-              <Icon name="exclamation-triangle" className="muted" size="sm" />
```

Comment:
this changes the icon's color:
Before:
<img width="475" alt="Screenshot 2022-07-22 at 13 11 59" src="https://user-images.githubusercontent.com/1170767/180436648-0a7836e5-034d-4ed1-9fe2-9e922e52d36a.png">

after:
<img width="488" alt="Screenshot 2022-07-22 at 13 12 19" src="https://user-images.githubusercontent.com/1170767/180436700-41516db0-8e83-49be-b456-ad81c4529607.png">

do we want this?

Reply:
Thanks @staton-hyse11.
I will remove the icon.